src/sentry/static/sentry/app/components/organizations/multipleEnvironmentSelector.tsx

```diff
@@ -193,19 +194,21 @@ class MultipleEnvironmentSelector extends React.PureComponent<Props, State> {
 
   getEnvironments() {
     const {projects, selectedProjects} = this.props;
+    const config = ConfigStore.getConfig();
```

Comment:
I think you can also call `get('user')` to get the user.

Reply:
Do you want me to change it before merging it?
Or is simply so I know for the next time?

Reply:
The latter